src/dcp/dcp.c

```diff
@@ -81,7 +87,8 @@ static int daos_check_args(
     int rank,
     char** argpaths,
     daos_args_t* da,
-    int* flag_daos_args)
+    int* flag_daos_args,
+    bool* is_posix_copy)
```

Comment:
To be consistent with some other tools that have DAOS integration, I wonder if we want to use something like `API in (POSIX, DAOS)` instead of `is_posix_copy`
For example, `ior` has these options:
```
-a POSIX # Assumes path is POSIX
-a DAOS # Uses the daos_ API
-a DFS # Uses the dfs_ API
```
For our purposes here, we can probably just use:
```
api = POSIX # POSIX paths, or POSIX containers with dfs_ API
api = DAOS # DAOS container with daos_ api
```

Reply:
@dsikich @adammoody Thoughts here?